### ISSUE TITLE
[PowerShell only] Auto hide PS window

### ISSUE THREAD

**Issue by htcfreek** [htcfreek/SimpleLapsGui#4]
_Same as the title._

**Comment by F1reF0x** [comment]
A possible implementation you can find at [https://gist.github.com/F1reF0x/abbd05668719b5670f07736b1eb5ac5f.js](url)

**Comment by htcfreek** [comment]
@F1reF0x 
Can you post the auto hide code here? Can't access the link.

**Comment by F1reF0x** [comment]
```
# .Net methods for hiding/showing the console in the background
# Usage: show-console -show / show-console -hide
function Show-Console
{
    param ([Switch]$Show,[Switch]$Hide)
    if (-not ("Console.Window" -as [type])) { 

        Add-Type -Name Window -Namespace Console -MemberDefinition '
        [DllImport("Kernel32.dll")]
        public static extern IntPtr GetConsoleWindow();

        [DllImport("user32.dll")]
        public static extern bool ShowWindow(IntPtr hWnd, Int32 nCmdShow);
        '
    }

    if ($Show)
    {
        $consolePtr = [Console.Window]::GetConsoleWindow()

        # Hide = 0,
        # ShowNormal = 1,
        # ShowMinimized = 2,
        # ShowMaximized = 3,
        # Maximize = 3,
        # ShowNormalNoActivate = 4,
        # Show = 5,
        # Minimize = 6,
        # ShowMinNoActivate = 7,
        # ShowNoActivate = 8,
        # Restore = 9,
        # ShowDefault = 10,
        # ForceMinimized = 11

        $null = [Console.Window]::ShowWindow($consolePtr, 5)
    }
    
    if ($Hide)
    {
        $consolePtr = [Console.Window]::GetConsoleWindow()
        #0 hide
        $null = [Console.Window]::ShowWindow($consolePtr, 0)
    }
}
```
Before showing the Dialog just
`show-console -hide`

When you start the powershell script: before the Dialog is shown, the console window will be hidden. After ShowDialog() you can:
`show-console -show`

**Comment by htcfreek** [comment]
Fixed and fix released with version `1.2`.
